suite/tests/client-interface/stolen-reg.dll.c

```diff
@@ -94,11 +99,11 @@ read_and_restore_stolen_reg_value()
     dr_get_mcontext(drcontext, &mc);
 
     /* The key part of the test: that the modified value shows up here. */
-    DR_ASSERT(mc.IF_ARM_ELSE(r10, r28) == test_value);
+    DR_ASSERT(mc.IF_ARM_ELSE(r10, IF_AARCH64_ELSE(r28, s11)) == test_value);
```

Comment:
Have a single define at the top of the file to avoid repeating this multi-condition sequence?